### PR TITLE
feat: new distro v1.23.2 sa issue new versioning

### PR DESCRIPTION
**Pull request by nutellinoit**

### CODE REVIEW

File: docs/releases/v1.23.2.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  - No updates
+- [OPA](https://github.com/sighupio/fury-kubernetes-opa) 📦 core module: v1.6.2 -> [**v1.6.2**](https://github.com/sighupio/fury-kubernetes-opa/releases/tag/v1.6.2)
+  - No updates
+
+> Please refer the individual release notes for detailed information
+
+## Upgrade path
+
+### Katalog Procedure
+To upgrade the distribution from `v1.23.1` to `v1.23.2` please follow the instructions written in the release notes of each core module.
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:39]: I don't think there are specific steps to perform, there are no breaking changes nor manual tasks do to AFAIK to upgrade from `v1.23.1` to `v1.23.2`.

Am I missing something?

**Review by ralgozino — CHANGES_REQUESTED** [review]

File: docs/releases/v1.23.2.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+## New Features
+
+### Core Module Updates
+
+- Removed `commonLabels` from all the `kustomize` katalogs
+
+- [networking](https://github.com/sighupio/fury-kubernetes-networking) 📦 core module: v1.8.2 -> [**v1.9.0**](https://github.com/sighupio/fury-kubernetes-networking/releases/tag/v1.9.0)
+  - Update [Calico] from version `3.21.3` to `3.23.2`.
+- [monitoring](https://github.com/sighupio/fury-kubernetes-monitoring) 📦 core module: v1.14.1 -> [**v1.14.2**](https://github.com/sighupio/fury-kubernetes-monitoring/releases/tag/v1.14.2)
+  - Bugfix on [x509-exporter]
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:24]:
```suggestion
  - Bugfix on [x509-exporter]
```

File: docs/releases/v1.23.2.md (hunk 1 above)
Comment (nutellinoit) [reply docs/releases/v1.23.2.md]: I removed the sentence

**Comment by nutellinoit** [comment]
This PR should be good to go

File: docs/releases/v1.23.2.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Kubernetes Fury Distribution Release v1.23.2
+
+Welcome to the KFD release `v1.23.2`. This is a patch release
+fixing bugs in all the core modules.
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:4]:
```suggestion
Welcome to the KFD release `v1.23.2`. This is a patch release fixing bugs in all the core modules.
```

File: docs/releases/v1.23.2.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The team has been working to make the release upgrade as simple as possible, so read carefully the upgrade path of each
+core module listed below along with the upgrade path of the distribution.
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:7]:
```suggestion
The team has been working to make the release upgrade as simple as possible, so read carefully the upgrade path of each core module listed below along with the upgrade path of the distribution.
```

File: docs/releases/v1.23.2.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+This distribution is maintained with ❤️ by the team [SIGHUP](https://sighup.io/),
+and is battle tested in production environments.
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:10]:
```suggestion
This distribution is maintained with ❤️ by the team [SIGHUP](https://sighup.io/), and is battle tested in production environments.
```

File: docs/releases/v1.23.2.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  - No updates
+- [OPA](https://github.com/sighupio/fury-kubernetes-opa) 📦 core module: v1.6.2 -> [**v1.6.2**](https://github.com/sighupio/fury-kubernetes-opa/releases/tag/v1.6.2)
+  - No updates
+
+> Please refer the individual release notes for detailed information
+
+## Upgrade path
+
+From this version, we are introducing the new versioning system, see the [versioning documentation file][versioning] to know more about
+the new versioning scheme of the distribution and the upgrade path.
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:34]:
```suggestion
From this version, we are introducing the new versioning system, see the [versioning documentation file][versioning] to know more about the new versioning scheme of the distribution and the upgrade path.
```

File: docs/releases/v1.23.2.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+### Katalog Procedure
+
+To upgrade the distribution from `v1.22.x` to `v1.23.2`, you need to download this new version, vendor the dependencies,
+finally applying the `kustomize` project.
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:39]:
```suggestion
To upgrade the distribution from `v1.22.x` to `v1.23.2`, you need to download this new version, vendor the dependencies, finally applying the `kustomize` project.
```

File: docs/releases/v1.23.2.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+```bash
+furyctl vendor -H
+kustomize build . | kubectl apply -f -
+```
+
+> **NOTE**: *The upgrade takes some minutes (depends on the cluster size), and you should expect some downtime during
+the upgrade process.*
```
Comment (ralgozino) [inline docs/releases/v1.23.2.md:47]:
```suggestion
> **NOTE**: *The upgrade takes some minutes (depends on the cluster size), and you should expect some downtime during the upgrade process.*
```

**Review by ralgozino — CHANGES_REQUESTED** [review]
minor styling changes

**Review by ralgozino — APPROVED** [review]
lgtm

**Pull request merged**